WidgetExtension/Variants/MultiFollowersCount/MultiFollowersCountWidget.swift

```diff
@@ -0,0 +1,163 @@
+// Copyright © 2023 Mastodon gGmbH. All rights reserved.
+
+import WidgetKit
+import SwiftUI
+import Intents
+import MastodonSDK
+import MastodonLocalization
+
+struct MultiFollowersCountWidgetProvider: IntentTimelineProvider {
+    func placeholder(in context: Context) -> MultiFollowersCountEntry {
+        .placeholder
+    }
+
+    func getSnapshot(for configuration: MultiFollowersCountIntent, in context: Context, completion: @escaping (MultiFollowersCountEntry) -> ()) {
+        loadCurrentEntry(for: configuration, in: context, completion: completion)
+    }
+
+    func getTimeline(for configuration: MultiFollowersCountIntent, in context: Context, completion: @escaping (Timeline<MultiFollowersCountEntry>) -> ()) {
+        loadCurrentEntry(for: configuration, in: context) { entry in
+            completion(Timeline(entries: [entry], policy: .after(.now)))
+        }
+    }
+}
+
+struct MultiFollowersCountEntry: TimelineEntry {
+    let date: Date
+    let accounts: [MultiFollowersEntryAccountable]?
+    let configuration: MultiFollowersCountIntent
+    
+    static var placeholder: Self {
+        MultiFollowersCountEntry(
+            date: .now,
+            accounts: [
+                MultiFollowersEntryAccount(
+                    followersCount: 99_900,
+                    displayNameWithFallback: "Mastodon",
+                    acct: "mastodon",
+                    avatarImage: UIImage(named: "missingAvatar")!,
+                    domain: "mastodon"
+                )
+            ],
+            configuration: MultiFollowersCountIntent()
+        )
+    }
+    
+    static var unconfigured: Self {
+        MultiFollowersCountEntry(
+            date: .now,
+            accounts: nil,
+            configuration: MultiFollowersCountIntent()
+        )
+    }
+}
+
+struct MultiFollowersCountWidget: Widget {
+    private var availableFamilies: [WidgetFamily] {
+        return [.systemSmall, .systemMedium]
+    }
+
+    var body: some WidgetConfiguration {
+        IntentConfiguration(kind: "Multiple followers", intent: MultiFollowersCountIntent.self, provider: MultiFollowersCountWidgetProvider()) { entry in
+            MultiFollowersCountWidgetView(entry: entry)
+        }
+        .configurationDisplayName("Multiple followers")
+        .description("Show number of followers for multiple accounts.")
```

Comment:
Can we localize this or is it not required?

Reply:
Oops, must have missed this, I've translated the rest, so yes we should definitely translate this as well. Will do now.